soak_bench_scripts/soak.sh

```diff
@@ -17,4 +19,4 @@ printf "Starting soak tests...\n"
 
 # Running fromm parsec folder - will need full path
 # This command will loop the test forever until it hits an error
-loop -q target/release/integration_tests-* | tee -a $LOGFILE
+loop -q target/release/integration_tests-* |& tee -a -i --output-error=warn $LOGFILE
```

Comment:
Btw, you may already know this, but I think that `--output-error` flag will only capture any errors for when `tee` tries to write somewhere that's problematic, but it won't give you anything extra from what went wrong with the `cargo` process.